src/TropicalGeometry/TropicalPolyhedron/constructors.jl

```diff
@@ -0,0 +1,151 @@
+const MinOrMax = Union{typeof(min), typeof(max)}
+
+struct TropicalPolyhedron{M<:MinOrMax} <: PolyhedralObject{QQFieldElem}
+  pm_tpolytope::Polymake.BigObject
+
+  @doc raw"""
+      TropicalPolyhedron{M}(p::Polymake.BigObject) where M<:MinOrMax
+
+  Constructs a `TropicalPolyhedron` corresponding to a `Polymake.BigObject` with tropical addition `M`.
+  """
```

Comment:
We should not document this constructor (it's internal). Instead there should be a docstring on the `struct` definition itself (so that users can find out what this struct is doing, and ideally also what the proper way for constructing an instance is: e.g. by proving a code example that use suitable functions (`tropical_convex_hull` I am guessing would be one?), or with `@ref`